### PR TITLE
Removed obsolete ci

### PR DESCRIPTION
**Pull request by Kostronor**

### CODE REVIEW

**Comment by Kostronor** [comment]
:shipit: @cltnschlosser @Raycoms

File: .travis.yml [hunk 1]
```diff
@@ -14,12 +11,3 @@ cache:
   - .gradle
   - $HOME/.gradle/caches/
   - $HOME/.gradle/wrapper/
-deploy:
```
Comment (cltnschlosser) [inline .travis.yml]: Removing this because its handled by the dropbox script?

**Comment by cltnschlosser** [comment]
:shipit:

**Comment by Raycoms** [comment]
:shipit:

File: .travis.yml (hunk 1 above)
Comment (Kostronor) [inline .travis.yml]: Exactly

**Pull request merged**